models/Document/Editable/Date.php

```diff
@@ -74,19 +74,21 @@ public function getDataEditmode()
      */
     public function frontend()
     {
+        if (!$this->date instanceof \DateTimeInterface) {
+            return null;
+        }
+
         $format = null;
 
         if (isset($this->config['outputFormat']) && $this->config['outputFormat']) {
             $format = $this->config['outputFormat'];
         } elseif (isset($this->config['format']) && $this->config['format']) {
             $format = $this->config['format'];
         } else {
-            $format = 'Y-m-d\TH:i:sO'; // ISO8601
```

Comment:
The old implementation displayed a literal `Y-m-d\TH:i:sO` in the browser, as the `formatLocalized` method expected the `%Y-%m...` notation.